vite.js

```diff
@@ -1,27 +1,30 @@
 const path = require('path')
 const fs = require('fs')
 
-module.exports = function proxyOptions({ port = 8080 } = {}) {
+module.exports = function proxyOptions({
+  port = 8080,
+  regex = '^/(app|login|api|assets|files)',
```

Comment:
```suggestion
  source = '^/(app|login|api|assets|files)',
```
Variable name "source" sounds more apt? 
like "all these sources are redirected to a different target"